extension/chrome/settings/modules/compatibility.ts

```diff
@@ -93,9 +93,8 @@ View.run(class CompatibilityView extends View {
         this.appendResult(`${skn} Created: ${await this.test(async () => this.formatDate(subKey.keyPacket.created))}`);
         this.appendResult(`${skn} Algo: ${await this.test(async () => `${subKey.getAlgorithmInfo().algorithm}`)}`);
         this.appendResult(`${skn} Verify: ${await this.test(async () => {
-          const verifyResNum = await subKey.verify(key.primaryKey);
-          const veryfyResWord = opgp.enums.read(opgp.enums.keyStatus, 1);
-          return `${verifyResNum}: ${veryfyResWord}`;
+          await subKey.verify(key.primaryKey);
```

Comment:
I figured the new OpenPGP.js api thorws on verify? In this case this was an issue even before this PR.

Reply:
Yes, this was an issue before this PR. I tested that on stable Flowcrypt from Chrome Web Store and the result was the same. I'm not sure which version introduced this bug. (It wasn't very prominent so I'm not surprised that this slipped by. I actually glanced at the compliance output and didn't notice there was a failure :D ).